.circleci/config.yml

```diff
@@ -58,36 +58,36 @@ jobs:
             set -x
             ./bin/deploy.sh dev
 
-  deploy_staging:
-    machine: true
-    steps:
-      - checkout
-      - run:
-          name: Build docker images
-          command: |
-            set -x
-            cp .env.test .env
-            docker-compose build
-      - run:
-          name: Run unit tests for API
-          command: |
-            set -x
-            docker-compose run api ./test.sh
-      - run:
-          name: Run unit tests for the front end
-          command: |
-            set -x
-            docker-compose run frontend npm test
-      - run:
-          name: Code coverage
-          command: |
-            set -x
-            echo CODECOV_TOKEN=$CODECOV_TOKEN >> .env
-            docker-compose run frontend npm run coverage
-      - deploy:
-          command: |
-            set -x
-            ./bin/deploy.sh staging
+  # deploy_staging:
```

Comment:
We've currently disabled the "staging" environment in CircleCI.  This is a side-effect of TrueTandem and 18F using separate github repositories.  We're both merging to `master`, and Circle is hooked into the same codebase both in the 18F and the TT repo.    

We tossed around the idea of merging repositories, which I think is still a good one.  In the meantime this seems to be the easiest way to get TT/master -> cloud.gov/staging, and 18F/master -> cloud.gov/production.

Instead of targeting different cloud.gov environments based on git branches we are looking at the github repo owner.  This was easier in Travis but a little clunky in Circle atm.  Here's the bit of code that tells the rest of the story: https://github.com/18F/e-QIP-prototype/pull/102/files#diff-01dd223246b3d82a911250aceaac85b5R18